ledger-tool/src/main.rs

```diff
@@ -1071,10 +1071,16 @@ fn load_bank_forks(
     }
 
     let account_paths = if let Some(account_paths) = arg_matches.value_of("account_paths") {
-        if !blockstore.is_primary_access() {
-            // Be defensive, when default account dir is explicitly specified, it's still possible
-            // to wipe the dir possibly shared by the running validator!
-            eprintln!("Error: custom accounts path is not supported under secondary access");
+        let allow_custom_accounts_path = arg_matches.is_present("allow_custom_accounts_path");
+        if !blockstore.is_primary_access() && !allow_custom_accounts_path {
+            // Be defensive, if we don't have primary access, the validator
+            // could be running and we don't want to disrupt that. Only allow
+            // custom path(s) if the flag specified to allow it.
+            eprintln!(
+                "Error: custom accounts path is not allowed under secondary access. \
```

Comment:
nit: `custom accounts path is not allowed under secondary access.` -> `custom accounts path by default is not allowed under secondary access.`